app/views/teacher_mailer/references_reminder.text.erb

```diff
@@ -0,0 +1,18 @@
+Dear <%= application_form_full_name(@application_form) %>
+
+We’re still waiting for a response from one or more of the references you provided to verify your work history.
+
+<% if @teacher.reminder_emails.count == 0 %>
```

Comment:
It looks like we define a `@number_of_reminders_sent` variable in the mailer but then don't use it in the view.